cmd/minishift/cmd/config/util.go

```diff
@@ -70,3 +71,9 @@ func SetBool(m MinishiftConfig, name string, val string) error {
 	m[name] = b
 	return nil
 }
+
+func SetSlice(m MinishiftConfig, name string, val string) error {
```

Comment:
I guess this is obsolete for this pull request, right? You skipped the slice based parameters for now, no?